src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -24,8 +24,9 @@ public class AddCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
             + "Parameters: " + PREFIX_NAME + "NAME " + PREFIX_PHONE + "PHONE " + "[" + PREFIX_EMAIL
-            + "EMAIL] " + "[" + PREFIX_ADDRESS + "ADDRESS] " + "[" + PREFIX_TAG + "TAG]... "
-            + PREFIX_DATEOFLASTVISIT + "DATEOFLASTVISIT " + PREFIX_EMERGENCY_CONTACT + "EMERGENCY_CONTACT \n"
+            + "EMAIL] " + "[" + PREFIX_ADDRESS + "ADDRESS] " + "[" + PREFIX_TAG + "TAG] "
```

Comment:
Are we still leaving the '...' behind tag? Because the '...' indicates that there can be multiple prefixes for it. 
For eg. t/friend t/team member t/buddy

Reply:
I'll add that back in for `t/` prefix